uint32 robot_id
---
uint32[] pose_ids
nav_msgs/Path path